docs/apache-airflow/installation/installing-from-pypi.rst

```diff
@@ -53,20 +53,28 @@ This is an example, see further for more explanation.
 Constraints files
 '''''''''''''''''
 
-Airflow installation can be tricky sometimes because Airflow is both a library and an application.
+Why do we need constraints
```

Comment:
The below suggestion is more of a statement so it doesn't need a `?` at the end.
```suggestion
Why we need constraints
```

Reply:
yep